contact_info_service.py

```
@@ -131,47 +135,70 @@ def get_contact_info(self, business_name: str):
+            except json.decoder.JSONDecodeError as e:
+                span.set_attribute("lookup.success", False)
+                span.set_attribute("error.type", "json_decode_error")
+                span.set_status(Status(StatusCode.ERROR))
+                span.record_exception(e)
+
+                raise ContactInfoServiceError(
+                    f"Invalid JSON response for '{business_name}': {e}"
+                ) from e
+
+            if "error" in data:
+                span.set_attribute("lookup.success", False)
```

Ruff (0.14.2)

Boolean positional value in function call

(FBT003)